.github/workflows/increment-version-on-merge.yml

```diff
@@ -38,13 +38,21 @@ jobs:
         with:
           ref: main
           fetch-depth: 0
-          persist-credentials: false
 
       # See https://github.com/im-open/git-version-lite for more details around how to increment
       # major/minor/patch through commit messages
       - name: Increment the version
-        uses: im-open/git-version-lite@main
+        id: version
+        uses: ./
         with:
-          create-ref: true
           github-token: ${{ secrets.GITHUB_TOKEN }}
           default-release-type: major
+          
+      - name: Create Version and Major Tags
```

Comment:
Why use a run step versus the action that was moved to im-open?

Reply:
Because it's simpler. There's no external dependency to keep up to date, or to wonder whether it's behaving properly. To me, it's abundantly clear what these git commands are doing. That other action was pretty complicated code to do the same thing as this, besides it has a hidden dependency on a previously-created release.

I like the saying "instead of writing code that has no obvious bugs, write code that's so simple it obviously has no bugs."

Reply:
I had to go back to the original PR to get the context for that.  I didn't know why the change was made.